dashboard/app/assets/stylesheets/application.scss

```diff
@@ -3256,6 +3256,10 @@ code {
   }
 }
 
+.breadcrumbHeader {
```

Comment:
Looks like we usually use kebab-case in this file?

Reply:
Yeah looks like you are right. I will fix this up